feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/TimetableItemDetailViewModel.kt

```diff
@@ -52,9 +54,14 @@ class TimetableItemDetailViewModel @Inject constructor(
             )
     private val viewBookmarkListRequestStateFlow =
         MutableStateFlow<ViewBookmarkListRequestState>(ViewBookmarkListRequestState.NotRequested)
+    private val currentDescriptionLanguage = MutableStateFlow(defaultLang())
```

Comment:
Initially, I was hoping that the session would be displayed in its original default language. The reason is that it would reflect what the speaker actually wrote. 👀 
Maybe you can have userSelectedDescriptionLanguage = MutableStateFlow(null)